# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1270186319 -7200
#      Fri Apr 02 07:31:59 2010 +0200
# Node ID 7b5f706f3a83a54046c9f7ee790ab9f6d1b320d1
# Parent  2645a6b1027b72afe6e812d523c6faf54332d135
typo in arma_rnd, by M. Vinyals

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-04-02  Marc Vinyals  <mrc_timer@users.sourceforge.net>
+
+	* signal/arma_rnd.m: fix typo in test.
+
 2010-03-30  Jaroslav Hajek  <highegg@gmail.com>
 
 	* specfun/factor.m: Use rem to check divisibility.
 
 2010-03-27  Ben Abbott <bpabbot@mac.com>
 
 	* plot/__go_draw_figure__.m: Untabify.
 	* plot/print.m: Provide default suffixes & validate hardcopy format.
diff --git a/scripts/signal/arma_rnd.m b/scripts/signal/arma_rnd.m
--- a/scripts/signal/arma_rnd.m
+++ b/scripts/signal/arma_rnd.m
@@ -42,17 +42,17 @@
 ## Author: FL <Friedrich.Leisch@ci.tuwien.ac.at>
 ## Description: Simulate an ARMA process
 
 function x = arma_rnd (a, b, v, t, n)
 
   if (nargin == 4)
     n = 100;
   elseif (nargin == 5)
-    if (!isscalar (t))
+    if (!isscalar (n))
       error ("arma_rnd: n must be a scalar");
     endif
   else
     print_usage ();
   endif
 
   if ((min (size (a)) > 1) || (min (size (b)) > 1))
     error ("arma_rnd: a and b must not be matrices");
